docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+
+|             Type            |     Default    |
+| :-------------------------: | :------------: |
+|  boolean **OR** enum above  | 'bottom-right' |
+
+---
+
+### `prevTitle`
+
+Prev button title
+
+|   Type   | Default |
+| :------: | :-----: |
+|  string  | 'Prev'  |
+
+---
+
+### `nextTitle`
+
+Next button title
+
+|   Type   | Default |
+| :------: | :-----: |
+|  string  | 'Prev'  |
+
+---
+
+### `dotsTouchable`
+
+Touches over dots will move swiper to relative slide
+
+|  Type   | Default |
+| :-----: | :-----: |
+| boolean |  false  |
+
+---
+
+### `dotsWrapperStyle`
+
+Dots wrapper View style
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `dotProps`
+
+Customizing dot with [Badge](/react-native-elements/docs/badge.html) props
+
+|                                      Type                                         | Default |
+| :-------------------------------------------------------------------------------: | :-----: |
+| {[...Badge props](/react-native-elements/docs/badge.html#props)} |  none   |
+
+---
+
+### `dotActiveStyle`
+
+Additional style to active dot
+
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
+
+---
+
+### `DotComponent`
+
+Custom dot component
+
+|   Type    |                              Default                             |
+| :-------: | :--------------------------------------------------------------: |
+| component | [Badge](/react-native-elements/docs/badge.html) |
+
+---
+
+### `prevProps`
+
+Customize prev button with [Button](/react-native-elements/docs/button.html) props
+
+|                                        Type                                         | Default |
+| :---------------------------------------------------------------------------------: | :-----: |
+| {[...Button props](/react-native-elements/docs/button.html#props)} |  none   |
+
+---
+
+### `nextProps`
+
+Customize next button with [Button](/react-native-elements/docs/button.html) props
+
+|                                        Type                                         | Default |
+| :---------------------------------------------------------------------------------: | :-----: |
+| {[...Button props](/react-native-elements/docs/button.html#props)} |  none   |
+
+---
+
+### `PrevComponent`
+
+Custom prev button component
+
+|   Type    |                               Default                              |
+| :-------: | :----------------------------------------------------------------: |
+| component | [Button](/react-native-elements/docs/button.html) |
+
+---
+
+### `NextComponent`
+
+Custom next button component
+
+|   Type    |                               Default                              |
+| :-------: | :----------------------------------------------------------------: |
+| component | [Button](/react-native-elements/docs/button.html) |
+
+---
+
+### `firstPrevElement`
```

Comment:
I'd say leave out this and `lastNextElement`.  It's very unlikely this will be used

Reply:
This is [from community](https://github.com/oxyii/react-native-web-swiper/issues/1)